src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+| `SSL_KEY`     | Mount path to the SSL key for the LMS.                     |
+| `CA_TLS_CERT` | Mount path to the TLS CA certificate for the orchestrator. |
+| `TLS_CERT`    | Mount path to the TLS certificate for the orchestrator.    |
+| `TLS_KEY`     | Mount path to the TLS key for the orchestrator.            |
+
+{{site.data.alerts.callout_success}}
+Cockroach Labs recommends mounting certificates to `/app/certs`.
+{{site.data.alerts.end}}
+
+#### Service type
+
+~~~ yaml
+lms:
+  service:
+    type: ClusterIP
+    port: 9043
+    metricsPort: 9044
+...
+orchestrator:
+  service:
+    type: ClusterIP
+    port: 4200
+    metricsPort: 4201
+~~~
+
+`service` specifies the [Kubernetes service type](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and ports for the LMS instances and orchestrator.
+
+#### Prometheus Operator
+
+~~~ yaml
+serviceMonitor:
+  enabled: false
+  labels: {}
+  annotations: {}
+  interval: 30s
+  namespaced: false
+~~~
+
+`serviceMonitor` is a custom resource used with the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/tree/main) for monitoring Kubernetes. For more information, see the [Prometheus Operator documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md).
+
+## Security
+
+Cockroach Labs recommends the following:
+
+- Manage your LMS configuration in an [external Kubernetes secret](#manage-external-secret).
+- To establish secure connections between the LMS pods and with your client, set up TLS certificates for the [LMS](#configure-lms-certificate), [source database and CockroachDB](#configure-source-and-target-certificates), and [orchestrator and client](#configure-orchestrator-and-client-certificates).
+
+#### Manage external secret
+
+Use an external secrets manager such as [External Secrets Operator](https://external-secrets.io/latest/) to create and manage a Kubernetes secret containing your LMS configuration. For information on Kubernetes secrets, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+In the [LMS configuration](#configuration), specify the name of your external secret with `configSecretName`:
+
+~~~ yaml
+lms:
+  configSecretName: "secret-name"
+...
+orchestrator:
+  configSecretName: "secret-name"
+~~~
+
+WIthin the secret, set LMS config keys inside a JSON object named `config.json`. For example, in an `ExternalSecret` named `lms-config` created with External Secrets Operator:
+
+~~~ yaml
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: lms-config
+spec:
+  ...
+  target:
+    name: lms-config
+    ...
+      data:
+        config.json: |
+          {
+            "INIT_SOURCE": "mysql://{username}:{password}@{protocol}({host}:{port})/{database}?sslmode=verify-full?sslrootcert=path/to/mysql.ca&sslcert=path/to/mysql.crt&sslkey=path/to/mysql.key",
```

Comment:
I recommend showing two blocks here: 
1. Parametrized version that index properties from the external secret that is pulled (how customers should do it. (i.e. `"SSL_CA": lms-config.ca). Or however the parametrized version looks.
2. The example you have that shows the hardcoded values.

Reply:
Specifically for the critical keys like init_source and init_target. 
The paths to the files being hard coded im not as concerned about. Also they need the file name at the end not just `/app/certs`